src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/config/ServiceBusCreateUpdatesReceiverConf.java

```diff
@@ -15,28 +15,29 @@
 
 @AutoConfigureAfter(ServiceBusSenderConfiguration.class)
 @Configuration
+@SuppressWarnings("PMD")
 public class ServiceBusCreateUpdatesReceiverConf {
 
-    @PostConstruct()
-    public void registerMessageHandlers() throws InterruptedException, ServiceBusException {
-        createUpdatesListenClient.registerMessageHandler(
-            createUpdatesBusReceiverTask,
-            messageHandlerOptions,
-            createUpdatesListenExecutor
-        );
-    }
+    private final IQueueClient createUpdatesListenClient;
+
+    private final CreateUpdatesBusReceiverTask createUpdatesBusReceiverTask;
 
     private static final ExecutorService createUpdatesListenExecutor =
         Executors.newSingleThreadExecutor(r ->
-            new Thread(r, "create-updates-queue-listen")
+                                              new Thread(r, "create-updates-queue-listen")
```

Comment:
new Thread happily fits on the line above

Reply:
moved it to the line before